components/loadMoreButton/index.vue

```diff
@@ -4,6 +4,7 @@
       face="round"
       fill="neutral"
       extend
+      @click="loadMore()"
     >
       Load More
     </AeButton>
```

Comment:
this needs to be wrapped in a `<no-ssr>aebutton</no-ssr>` this should already be in there. i must have forgot to do this. That should solve the problem for now. also change the import to 
change this to 
`````
let AeButton

if (process.browser) {
  AeButton = require('@aeternity/aepp-components-3').AeButton
}
`````

Reply:
updated